SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -368,10 +378,16 @@ class TupleElementAddrInst : SingleValueInstruction, UnaryInstruction {
 }
 
 final public class StructInst : SingleValueInstruction {
+  public var uniqueNonTrivialFieldValue: Value? {
+    StructInst_getUniqueNonTrivialFieldValue(bridged).value
+  }
 }
 
 final public class StructExtractInst : SingleValueInstruction, UnaryInstruction {
   public var fieldIndex: Int { StructExtractInst_fieldIndex(bridged) }
+  public var isFieldOnlyNonTrivialField: Bool { 
```

Comment:
and here

Reply:
This one needs to access some AST types, like `TypeExpansionContext` and `VarDecl`. Should I bridge those types, or keep whole `StructExtractInst_isFieldOnlyNonTrivialField` bridged?